zucchini-ui-frontend/src/failures/components/FailuresTable.jsx

```diff
@@ -0,0 +1,148 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Link} from 'react-router';
+import Table from 'react-bootstrap/lib/Table';
+import Label from 'react-bootstrap/lib/Label';
+import Modal from 'react-bootstrap/lib/Modal';
+import Status from '../../ui/components/Status';
+
+export default class FailuresTable extends React.Component {
+
+  render() {
+    const {failures} = this.props;
+    const rows = failures.failures.map(groupedFailures => {
+      const nbFailedScenarii = groupedFailures.failedScenarii.length;
+      return (
+        groupedFailures.failedScenarii.map((failedScenario, index) => {
+          return <FailuresTableRow key={index} failedScenario={failedScenario} isFirstFailure={index === 0 } nbFailedScenarii={nbFailedScenarii}/>
+        })
+      );
+    });
+
+    return (
+      <Table bordered striped hover style={{tableLayout: 'fixed'}}>
+        <thead>
+          <tr>
+            <th className='col-md-4'>Erreur</th>
+            <th className='col-md-6'>Scénario</th>
+            <th className='col-md-1'>Statut</th>
+            <th className='col-md-1'>Analysé</th>
+          </tr>
+        </thead>
+        <tbody>{rows}</tbody>
+      </Table>
+    );
+  }
+
+}
+
+FailuresTable.propTypes = {
```

Comment:
Integrate `propTypes` in a static field in class `FailuresTable` : there is a syntax extension by Babel that supports static properties in Javascript classes.